server/@types/express/index.d.ts

```diff
@@ -10,7 +10,9 @@ declare module 'express-session' {
     nowInMinutes: number
 
     booker: Booker
-    bookingJourney: BookingJourney
+
+    bookingJourney?: BookingJourney
+    bookingConfirmed?: BookingConfirmed
```

Comment:
New `bookingConfirmed` item in session so we can delete the `bookingJourney` at the end but hold on to what we need for the booking confirmation page.